.gitignore

```diff
@@ -1 +1,6 @@
 /bin/
+terraform.*.*
```

Comment:
Please remove these ignore, we don't want to add resources created as a result of installation to be ignored. Use `--dir` flag if you would like to keep them separate, or use local git ignore (https://www.jerriepelser.com/blog/exlude-git-files-locally/) that is not controlled by installer public repo.

Reply:
done

Reply:
> done

I think this file should be deleted.

Reply:
Sorry, don't delete the file, but all changes need to be reverted.